src/jobsautoreport/main.py

```diff
@@ -43,7 +43,7 @@ def main() -> None:
                 "type": "section",
                 "text": {
                     "type": "mrkdwn",
-                    "text": f"• Jobs success rate: _{success_rate}_\n• Number of jobs triggered: _{number_of_jobs_triggered}_\n• Jobs average duration: _{average_job_duration}_",
+                    "text": f"• Jobs success rate: _{success_rate}_\n• Number of jobs triggered: _{number_of_jobs_triggered}_\n• Jobs average duration: _{average_jobs_duration.seconds // 60}_ minutes, and _{average_jobs_duration.seconds % 60}_ seconds",
```

Comment:
```suggestion
                    "text": f"• Jobs success rate: _{success_rate}_\n• Number of jobs triggered: _{number_of_jobs_triggered}_\n• Jobs average duration: _{average_jobs_duration.total_seconds() // 60}_ minutes, and _{average_jobs_duration.total_seconds() % 60}_ seconds",
```
`seconds` counts up to one day https://stackoverflow.com/a/51653034